src/components/form/form-actions.jsx

```diff
@@ -58,6 +79,24 @@ class FormActions extends React.Component {
     return (!this.props.isValid && this.props.errorMessage);
   }
 
+  /**
+   * Renders a warning that a favorite was changed and changes can be saved
+   * or discarded.
+   *
+   * @returns {React.Component}
+   */
+  renderUnsavedMessage() {
+    return (
+      <div>
+        You have unsaved changes. <a
+          onClick={this.onFavoriteChangeDiscarded}
```

Comment:
nit: i find this style of writing `<a>` tags super unreadable :(

Reply:
do you have suggestions? to keep links and text at the same line.